### PR TITLE
QA: Migrate About section to HTML

### PR DESCRIPTION
**Pull request by miguelangel-dev**
## Details
After the last QA session (4 March) we have decided to show in About section text enrichment using HTML. 

User interaction:
- Users can tap on the mails, phone numbers and links show in the text. 
- The text should be selectable.
- Links, emails, numbers should show a different appearance.

This PR will deprecate the last design, so the fixed button (near 47 disclaimers) won't be necessary anymore (it will be included in the HTML as a link)

Simulator Screen Shot - iPhone 11 - 2020-03-05 at 11 55 44

### CODE REVIEW

File: ScalaDays/Controllers/SDAboutViewController.xib [hunk 1]
```diff
@@ -24,41 +20,17 @@
...
-                            <exclude reference="RHr-cp-e44"/>
-                        </mask>
-                    </variation>
-                </scrollView>
+                <textView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" showsHorizontalScrollIndicator="NO" editable="NO" translatesAutoresizingMaskIntoConstraints="NO" id="avo-oW-dXi">
+                    <rect key="frame" x="0.0" y="44" width="414" height="774"/>
+                    <color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                    <string key="text">Lorem ipsum dolor sit er elit lamet, consectetaur cillium adipisicing pecu, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Nam liber te conscient to factor tum poen legum odioque civiuda.</string>
```
Comment (truizlop) [inline ScalaDays/Controllers/SDAboutViewController.xib]: I guess this is a temporary placeholder. We need to be careful not to ship with this text.

Comment (miguelangel-dev) [reply ScalaDays/Controllers/SDAboutViewController.xib]: Good catch! ;)

**Comment by MaureenElsberry** [comment]
What happened to all the "n" s in the code of conduct screen shot?!?

**Comment by miguelangel-dev** [comment]
> What happened to all the "n" s in the code of conduct screen shot?!?

Yep, you are right, we are receiving on this way from backend, Rafa is working on that. Thanks for keeping an eye on this! ^^

**Pull request merged**